src/main/java/VASSAL/build/module/PrototypesContainer.java

```diff
@@ -43,6 +44,10 @@
   private Map<String,PrototypeDefinition> definitions =
     new HashMap<>();
 
+  public Collection<PrototypeDefinition> getDefinitions() {
+    return definitions.values();
+  }
```

Comment:
Do we want the prototype defs mapping to be modifiable here? (It would be if we return `values()` instead of wrapping that in a `Collections.unmodifiableCollection()` before returning it.) I'm not saying either way, as I'm not familiar with this code.